frontend/demo/component/tabs/tabsheet-prefix-suffix.ts

```diff
@@ -0,0 +1,44 @@
+import 'Frontend/demo/init'; // hidden-source-line
+
+import { html, LitElement } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import '@vaadin/tabs';
+import '@vaadin/tabsheet';
+import '@vaadin/button';
+import '@vaadin/icon';
+import '@vaadin/icons';
```

Comment:
nit: let's sort imports alphabetically.

```suggestion
import '@vaadin/button';
import '@vaadin/icon';
import '@vaadin/icons';
import '@vaadin/tabs';
import '@vaadin/tabsheet';
```